ui/ui.go

```diff
@@ -118,12 +118,36 @@ func (m Model) Update(msg tea.Msg) (tea.Model, tea.Cmd) {
 			return m, cmd
 		}
 
+		if m.prSidebar.GetIsAssigning() {
```

Comment:
I think we can just merge with the condition above: `if m.prSidebar.GetIsAssigning() || m.prSidebar.GetisCommenting...`